test/libsolidity/syntaxTests/using/global_library_for_defined_elsewhere.sol

```diff
@@ -0,0 +1,6 @@
+using L for L.S global;
+library L {
+    struct S { uint x; }
+}
```

Comment:
Can you also add a positive case for this? I.e. is this supposed to work?
```solidity
using L for S global;
struct S { uint x; }
library L {}
```

Reply:
We already have a test: https://github.com/ethereum/solidity/blob/develop/test/libsolidity/semanticTests/using/using_global_library.sol

Reply:
ok. I mean in that example it's an UDVT not a struct but I guess they should be handled by the same code, right?